packages/core/test/unit/dialects/mysql/default-value-parser-internal.test.ts

```diff
@@ -0,0 +1,36 @@
+import { parseDefaultValue } from '@sequelize/core/_non-semver-use-at-your-own-risk_/dialects/mysql/default-value-parser-internal.js';
+import { expect } from 'chai';
+
+import * as Support from '../../../support';
+
+const dialect = Support.getTestDialect();
+
+if (dialect.startsWith('mysql')) {
+  describe('[MySQL Specific] parseDefaultValue', () => {
```

Comment:
You might already be aware of this, but we have a helper function called `expectPerDialect` which you can use to add different assertions based on the dialect you are testing on. We prefer that that is used instead of adding dialect-specific test suites. It's fine while you are working on this, but I just wanted to bring it to your attention before the initial review

Reply:
Thanks, I think I'll use that in the general integration test of `describeTable`.

I don't think that could be usefull in the situation of testing `default-value-parser` with unit test, because tested cases heavily depend on the dialect too.